src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/reports/ReportsService.java

```diff
@@ -59,6 +61,56 @@ public ReportsService(
         return reportResult;
     }
 
+    public List<EnvelopeCountSummaryReportItem> getEnvelopeCountSummaryReportItems(
+        List<EnvelopeCountSummary> result
+    ) {
+        return  result
+            .stream()
+            .map(item -> new EnvelopeCountSummaryReportItem(
+                item.received,
+                item.rejected,
+                item.container,
+                item.date
+            ))
+            .collect(toList());
+
+    }
+
+
+    public EnvelopeCountSummaryReportListResponse getCountSummaryResponse(
+        List<EnvelopeCountSummary> result
+    ) {
+        // Timestamp
+        String localDateTime = getTimeStamp();
+
+        // Total number of rejected Envelopes
+        int totalRejected = getTotalRejected(result);
+
+        // Total number of received Envelopes
+        int totalReceived = getTotalReceived(result);
```

Comment:
`totalRejected` and `totalReceived` should be derived in one iteration. Having two iterations across the same list is not optimal though the performance impact is minor..